### DISCUSSION TITLE
SuperTux v0.6.3-beta.2

### DISCUSSION THREAD

**Discussion by tobbi** [SuperTux/supertux#1981] [Announcements]
The SuperTuxTeam is happy to announce the second beta release for SuperTux 0.6.3.

## What's Changed
* Allow scrolling in credits screen with up/down keys by @martinvanzijl in https://github.com/SuperTux/supertux/pull/1402
* Fix issue with live fire not fading after colliding with bomb by @martinvanzijl in https://github.com/SuperTux/supertux/pull/1405
* Add missing releases to appdata file by @xduugu in https://github.com/SuperTux/supertux/pull/1409
* Remove link between coins and checkpoints by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1410
* Add swimming by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1407
* Reorganize grey brick tiles by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1411
* Override only configurable keys (partially fixes #1400) by @szborows in https://github.com/SuperTux/supertux/pull/1406
* Expand snow tileset by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1415
* Apply ice friction only when Tux toches the ground by @HybridDog in https://github.com/SuperTux/supertux/pull/1414
* remove extra space from function and parameter by @jgarte in https://github.com/SuperTux/supertux/pull/1427
* Add the RubLight object by @HybridDog in https://github.com/SuperTux/supertux/pull/1416
* Remember --spawnpoint and --sector across deaths. by @divVerent in https://github.com/SuperTux/supertux/pull/1419
* INSTALL.md: Add information about reducing compilation time by @HybridDog in https://github.com/SuperTux/supertux/pull/1425
* Editor: add a "test from here" option by @divVerent in https://github.com/SuperTux/supertux/pull/1423
* Add new haywire animation by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1429
* Crash fix: do not allow deleting the last node of a path. by @divVerent in https://github.com/SuperTux/supertux/pull/1432
* Crash fix: handle dragging a selection rectangle up/left correctly. by @divVerent in https://github.com/SuperTux/supertux/pull/1433
* When dragging tile selections upwards or to the left, fix offset math. by @divVerent in https://github.com/SuperTux/supertux/pull/1434
* Make the other Rectf ctor also assert that size can't be negative. by @divVerent in https://github.com/SuperTux/supertux/pull/1435
* First update editor widgets, then flush deleted objects away. by @divVerent in https://github.com/SuperTux/supertux/pull/1436
* Fix UBSan error at early startup about converting infinity to int. by @divVerent in https://github.com/SuperTux/supertux/pull/1438
* Add a "speed" property to path nodes. by @divVerent in https://github.com/SuperTux/supertux/pull/1424
* Fix assertion failure in triangle drawing on the SDL backend. by @divVerent in https://github.com/SuperTux/supertux/pull/1437
* Spelling: Fractions for tiles by @comradekingu in https://github.com/SuperTux/supertux/pull/765
* Add sideways icecrushers by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1431
* Improve flying snowball's hat design by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1443
* Minor snow jumpy adjustments by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1444
* Pull Request to change the Boat by swimming Tux by @AndrePinheiroPT in https://github.com/SuperTux/supertux/pull/1447
* Issue 1398 fix throwable bug with coins by @martinvanzijl in https://github.com/SuperTux/supertux/pull/1449
* Fix bug with cloud particles disappearing too soon by @martinvanzijl in https://github.com/SuperTux/supertux/pull/1457
* Level Improvements by @Alasdairbugs in https://github.com/SuperTux/supertux/pull/1461
* Level Improvements by @Alasdairbugs in https://github.com/SuperTux/supertux/pull/1463
* Level Improvements by @Alasdairbugs in https://github.com/SuperTux/supertux/pull/1464
* Level Improvements Again by @Alasdairbugs in https://github.com/SuperTux/supertux/pull/1466
* Fixed a bug where you could travel through magic blocks while holding… by @Semphriss in https://github.com/SuperTux/supertux/pull/1465
* Fix manual value changing in the colour menu by @HybridDog in https://github.com/SuperTux/supertux/pull/1456
* Snail stomping by @Zwatotem in https://github.com/SuperTux/supertux/pull/1337
* Adjust iceslick textures by @HybridDog in https://github.com/SuperTux/supertux/pull/1105
* Level Aesthetic Improvements by @Alasdairbugs in https://github.com/SuperTux/supertux/pull/1474
* Add Circular Pathed Platforms by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1467
* Add Falling Platform by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1469
* Fix animation issue with slow framerates by @martinvanzijl in https://github.com/SuperTux/supertux/pull/1481
* Add sideways bumpers by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1478
* Merged the particles directories by @erratic-c-programmer in https://github.com/SuperTux/supertux/pull/1484
* Rocks can activate buttons by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1489
* Dispensers can obey gravity by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1490
* Fixed transition between worlds 1 and 2 by @mbernard2 in https://github.com/SuperTux/supertux/pull/1496
* Wind: Affect Tux on ground + Fancy Particles Option by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1493
* Fixed undo manager to properly handle the undo stack by @Semphriss in https://github.com/SuperTux/supertux/pull/1504
* Added option for wind to not affect player (decorative wind) by @Semphriss in https://github.com/SuperTux/supertux/pull/1503
* Added Squirrel function to edit the action (animation) of a platform's sprite by @Semphriss in https://github.com/SuperTux/supertux/pull/1506
* Added fading gradients through Squirrel by @Semphriss in https://github.com/SuperTux/supertux/pull/1508
* Support for easing on any path object by @Semphriss in https://github.com/SuperTux/supertux/pull/1507
* Fix issue #1492 with the alternative steps + Fix issue #1383 by @omeritzics in https://github.com/SuperTux/supertux/pull/1520
* Autotiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1519
* Add Hebrew translation of SuperTux description by @omeritzics in https://github.com/SuperTux/supertux/pull/1521
* Expose decals to scripting by @Zwatotem in https://github.com/SuperTux/supertux/pull/1512
* Layout widget in editor is now scrollable by @Semphriss in https://github.com/SuperTux/supertux/pull/1528
* Support for skippable cutscenes by @Semphriss in https://github.com/SuperTux/supertux/pull/1513
* Discord RPC Integration by @Semphriss in https://github.com/SuperTux/supertux/pull/1530
* Updated Badguys (WIP) by @Alzter in https://github.com/SuperTux/supertux/pull/1327
* Make badguys' pattern not random-based (WIP) by @Semphriss in https://github.com/SuperTux/supertux/pull/1541
* Cheats in editor by @Semphriss in https://github.com/SuperTux/supertux/pull/1540
* Corner-based autotile system by @Semphriss in https://github.com/SuperTux/supertux/pull/1543
* Allow sprite animations to continue each other by @Semphriss in https://github.com/SuperTux/supertux/pull/1558
* CMakeLists.txt: compile squirrel with -fPIC by @ffontaine in https://github.com/SuperTux/supertux/pull/1561
* Coyote time by @Semphriss in https://github.com/SuperTux/supertux/pull/1563
* Fixed Squirrel inheritance/multiple exposure (Will O'Wisps are back in scripting) by @Semphriss in https://github.com/SuperTux/supertux/pull/1566
* Reset all profiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1565
* Support for zooming from scripting by @Semphriss in https://github.com/SuperTux/supertux/pull/1560
* Fixed several problems with the sound system by @Semphriss in https://github.com/SuperTux/supertux/pull/1569
* Editor autosave by @Semphriss in https://github.com/SuperTux/supertux/pull/1554
* Optimize tile selection rendering by @Semphriss in https://github.com/SuperTux/supertux/pull/1571
* Align selection by @Semphriss in https://github.com/SuperTux/supertux/pull/1572
* Add new bouncing snowball sprite by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1573
* Fix broken paths by @plonibarploni in https://github.com/SuperTux/supertux/pull/1576
* Remove extraneous font image causing path conflict by @eric135 in https://github.com/SuperTux/supertux/pull/1585
* don't spawn Tux in the ground by @plonibarploni in https://github.com/SuperTux/supertux/pull/1575
* Custom particles by @Semphriss in https://github.com/SuperTux/supertux/pull/1544
* Added ladders for worldmaps by @Semphriss in https://github.com/SuperTux/supertux/pull/1589
* Prevent player from falling off the screen while swimming by @Semphriss in https://github.com/SuperTux/supertux/pull/1598
* Demo custom particles by @Semphriss in https://github.com/SuperTux/supertux/pull/1594
* Fixed snow particles scripting + menu Christmas level by @Semphriss in https://github.com/SuperTux/supertux/pull/1600
* Added worldmap road forking by @Semphriss in https://github.com/SuperTux/supertux/pull/1599
* Refactor of the presence integration & Discord RPC by @Semphriss in https://github.com/SuperTux/supertux/pull/1588
* [WIP] Move to C++17 by @strikersh in https://github.com/SuperTux/supertux/pull/1602
* Update icons by @Semphriss in https://github.com/SuperTux/supertux/pull/1604
* Code cleanup by @Semphriss in https://github.com/SuperTux/supertux/pull/1606
* Particle editor improvements by @Semphriss in https://github.com/SuperTux/supertux/pull/1605
* supertux/main: add try-catch for locale initialization. by @ericonr in https://github.com/SuperTux/supertux/pull/1601
* More and better worldmap roads! by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1610
* Create github actions matrix build by @maths22 in https://github.com/SuperTux/supertux/pull/1613
* New crystalcave tileset by RustyBox and Alzter by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1615
* Actions Cleanup by @maths22 in https://github.com/SuperTux/supertux/pull/1616
* Added ladders to forest worldmap by @Semphriss in https://github.com/SuperTux/supertux/pull/1612
* Added Color (with alpha) to background objects by @Semphriss in https://github.com/SuperTux/supertux/pull/1621
* Changed build scripts: Only one source artifact, and ci skip support by @Semphriss in https://github.com/SuperTux/supertux/pull/1620
* Fixed bug where ScriptTrigger areas wouldn't save their name by @Semphriss in https://github.com/SuperTux/supertux/pull/1622
* Improved interface textbox control by @Semphriss in https://github.com/SuperTux/supertux/pull/1614
* In-game progress stats by @Semphriss in https://github.com/SuperTux/supertux/pull/1619
* Improved game speed scripting by @Semphriss in https://github.com/SuperTux/supertux/pull/1629
* Let's climb that mountain: crystal repair by @Zwatotem in https://github.com/SuperTux/supertux/pull/1632
* Fixed tile selection not rendering properly on tilemaps with an offset by @Semphriss in https://github.com/SuperTux/supertux/pull/1636
* Editor region/rectangle tool will no longer paint tiles until the mouse button is released by @Semphriss in https://github.com/SuperTux/supertux/pull/1634
* Editor Scroll Speed small tweaks by @Raoul1808 in https://github.com/SuperTux/supertux/pull/1611
* Code cleanup: Simplified cutscene text rendering by @Semphriss in https://github.com/SuperTux/supertux/pull/1639
* Wall Jumping by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1631
* New rock decoration and other misc. tiles by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1640
* Fixed tilemap tiles converted to objects by @Semphriss in https://github.com/SuperTux/supertux/pull/1625
* Replace value copying with std::move and const references by @HybridDog in https://github.com/SuperTux/supertux/pull/1642
* Added fade helpers to help clean code in game objects by @Semphriss in https://github.com/SuperTux/supertux/pull/1641
* Fix bag badguy animation by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1648
* Added FreeBSD workflow in GitHub Actions by @Semphriss in https://github.com/SuperTux/supertux/pull/1649
* Added support for getting/setting the player's position by @Semphriss in https://github.com/SuperTux/supertux/pull/1653
* Deletion of a reference to an old (and deleted) file about level editor by @hogren in https://github.com/SuperTux/supertux/pull/1647
* Added usage of skippable cutscenes in cutscene levels by @Semphriss in https://github.com/SuperTux/supertux/pull/1637
* Added acknowledgements, required by some third-party libraries by @Semphriss in https://github.com/SuperTux/supertux/pull/1628
* Rock tiles are no longer solid. FINALLY! by @Semphriss in https://github.com/SuperTux/supertux/pull/1609
* Code mistakes found with TscanCode by @HybridDog in https://github.com/SuperTux/supertux/pull/1617
* supertux/main: Only set global locale on Windows by @shawnanastasio in https://github.com/SuperTux/supertux/pull/1663
* Windowsdylan icecrash patch by @windowsdylan in https://github.com/SuperTux/supertux/pull/1658
* [QoL] Native cursor option by @Raoul1808 in https://github.com/SuperTux/supertux/pull/1666
* Cheat Menu: Set a high number of the bonus count e.g. the number of fire bullets by @HybridDog in https://github.com/SuperTux/supertux/pull/1665
* Expanding of the retro snow tileset by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1668
* The Revenge In Redmond Revamp by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1669
* [QoL] Climbable small tweak by @Raoul1808 in https://github.com/SuperTux/supertux/pull/1670
* Added Windows workflow by @Semphriss in https://github.com/SuperTux/supertux/pull/1673
* Added workflow for Ubuntu Touch (UBports) by @Semphriss in https://github.com/SuperTux/supertux/pull/1655
* Added workflow for 32-bit Linux (i386,x86) by @Semphriss in https://github.com/SuperTux/supertux/pull/1675
* Added complete scripting support for custom particles by @Semphriss in https://github.com/SuperTux/supertux/pull/1633
* Fixed issues with moving platforms by @mbernard2 in https://github.com/SuperTux/supertux/pull/1514
* Changed handling of Tux movement by @Semphriss in https://github.com/SuperTux/supertux/pull/1680
* Decals now automatically update their sprite while in editor by @Semphriss in https://github.com/SuperTux/supertux/pull/1686
* Segfault fix by @mbernard2 in https://github.com/SuperTux/supertux/pull/1697
* Level editor tile grid shadow and constant-width lines by @HybridDog in https://github.com/SuperTux/supertux/pull/1693
* Major improvements to UBports build by @Semphriss in https://github.com/SuperTux/supertux/pull/1705
* Fix for 32-bit CI builds by @Semphriss in https://github.com/SuperTux/supertux/pull/1701
* Fixed grabbind rusty trampoline segfault by @Semphriss in https://github.com/SuperTux/supertux/pull/1711
* Temporary fix to visually show blocked paths by @Semphriss in https://github.com/SuperTux/supertux/pull/1700
* Added color support for Will'O'Wisps by @Semphriss in https://github.com/SuperTux/supertux/pull/1708
* Skull tiles now respawn 10 seconds after crumbling by @Semphriss in https://github.com/SuperTux/supertux/pull/1713
* Fixed ice unstable block animation error by @Semphriss in https://github.com/SuperTux/supertux/pull/1720
* Replace external/googletest with external dependency by @Grumbel in https://github.com/SuperTux/supertux/pull/1731
* Added bonus lightboxes that are on by default by @Semphriss in https://github.com/SuperTux/supertux/pull/1683
* Added scripting to rotating spotlight object + some changes by @Semphriss in https://github.com/SuperTux/supertux/pull/1689
* Added new lightmap tiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1714
* Worldmap tux swimming animation no longer hardcoded by @Semphriss in https://github.com/SuperTux/supertux/pull/1690
* Swimming Improvements by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1704
* Attempt to fix swimming issues with UBports build by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1740
* Worldmap tileset expansion by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1702
* Update Worldmap of Bonus-Island IV to use Halloween Tiles by @serano01 in https://github.com/SuperTux/supertux/pull/1743
* Fix premature EOF in IFileStreambuf by @Grumbel in https://github.com/SuperTux/supertux/pull/1746
* Minor Fixes in Bonus-Island III by @serano01 in https://github.com/SuperTux/supertux/pull/1744
* WASM/Emscripten port (SuperTux in the browser) by @Semphriss in https://github.com/SuperTux/supertux/pull/1696
* Update freebsd CI to fix missing rsync issue by @Grumbel in https://github.com/SuperTux/supertux/pull/1748
* Replace Vector with glm::vec2 by @Grumbel in https://github.com/SuperTux/supertux/pull/1735
* Icy island timeshift by @Semphriss in https://github.com/SuperTux/supertux/pull/1692
* Fix glm check failing on Nix and EmuELEC by @Grumbel in https://github.com/SuperTux/supertux/pull/1753
* Fix segfault when quitting a level while grabbing by @seanyeh in https://github.com/SuperTux/supertux/pull/1758
* Additional information can now be specified for levels. by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1729
* Fix music not pausing when the game is paused by @yyefymov in https://github.com/SuperTux/supertux/pull/1759
* Fix the font problems of Hebrew and Devanagari by @omeritzics in https://github.com/SuperTux/supertux/pull/1762
* UI improvements by @Semphriss in https://github.com/SuperTux/supertux/pull/1662
* SuperTux has moved from Freenode to Libera Chat. by @pazkero in https://github.com/SuperTux/supertux/pull/1765
* Haywire Updates by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1757
* Rename ubports mk dir to clickable by @jonnius in https://github.com/SuperTux/supertux/pull/1766
* Fix action in editor when closing a dialog by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1778
* Minor Level Fixes by @serano01 in https://github.com/SuperTux/supertux/pull/1770
* Add GLM dependency to INSTALL.md by @Ordoviz in https://github.com/SuperTux/supertux/pull/1780
* Walljump Fixes and Improvements by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1756
* More lenient spike hitboxes by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1741
* Fix floating on push buttons by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1767
* Crystallo Improvements and Additions by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1783
* Fix for WASM builds by @Semphriss in https://github.com/SuperTux/supertux/pull/1794
* Cutscene audio fix by @Semphriss in https://github.com/SuperTux/supertux/pull/1795
* Fix backflips not rotating by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1796
* Improved loading screen for WASM by @Semphriss in https://github.com/SuperTux/supertux/pull/1797
* Linux support for vcpkg by @Semphriss in https://github.com/SuperTux/supertux/pull/1802
* Include <cfloat> for FLT_EPSILON by @jwakely in https://github.com/SuperTux/supertux/pull/1804
* Fix haywire crashing by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1805
* Remove sky from snow backgrounds by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1801
* Fix negative coins by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1791
* Updated menu graphics, logo and icon by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1800
* Minor Level Fixes by @serano01 in https://github.com/SuperTux/supertux/pull/1807
* Fix for hat sprite desync by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1755
* Fix Tux sliding from platform by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1814
* Editor: Add an OKLab colour picker by @HybridDog in https://github.com/SuperTux/supertux/pull/1816
* Fix text anchor points by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1815
* Fix fade animation by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1817
* Remove sky gradient from two more backgrounds by @Rusty-Box in https://github.com/SuperTux/supertux/pull/1818
* Revert "Snails can only be stomped by buttjumping" by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1812
* Fix issue when snail is changing direction by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1831
* Fixed crash when autosave fails by @Semphriss in https://github.com/SuperTux/supertux/pull/1836
* Fix Ubuntu Touch (clickable) builds by @tobbi in https://github.com/SuperTux/supertux/pull/1841
* Attempt to fix missing SDL.h with VCPKG builds by @Semphriss in https://github.com/SuperTux/supertux/pull/1827
* Move GLM_ENABLE_EXPERIMENTAL into compile flags by @Grumbel in https://github.com/SuperTux/supertux/pull/1813
* Fix snails not colliding with other snails by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1844
* Set default folder in music selection to /music by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1830
* fixing music not playing after death sometimes (#1833) by @LouriNoel in https://github.com/SuperTux/supertux/pull/1834
* Don't draw coins count when in editor by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1846
* Added RustyBox's new crystal tiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1848
* Fix coins in frozen blocks can be collected by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1847
* Cloud particles are now contained in the camera viewport by @Semphriss in https://github.com/SuperTux/supertux/pull/1709
* Fix not working unisolid tiles by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1842
* Bezier curves on paths by @Semphriss in https://github.com/SuperTux/supertux/pull/1624
* Fix windows maybe by @tobbi in https://github.com/SuperTux/supertux/pull/1862
* Wind objects now affect Rocks by @Semphriss in https://github.com/SuperTux/supertux/pull/1855
* Upload WASM build to website by @Semphriss in https://github.com/SuperTux/supertux/pull/1866
* Make WASM use IndexedDB for data persistence by @Semphriss in https://github.com/SuperTux/supertux/pull/1869
* Fixed WASM warnings and added Werror flag by @Semphriss in https://github.com/SuperTux/supertux/pull/1870
* Fix Windows nightly uploads by @tobbi in https://github.com/SuperTux/supertux/pull/1872
* Show world description at bottom of screen by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1876
* Use jump to enter menus by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1875
* When pressing ESC after dying at a level start, ignore the play time if left or right is hold by @HybridDog in https://github.com/SuperTux/supertux/pull/1852
* Added autotiles for forest tiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1877
* Bugfixes in the particle editor by @Semphriss in https://github.com/SuperTux/supertux/pull/1687
* Throw bomb like icecube by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1742
* Black Will-o-Wisps will now extinguish a lit lantern by @Semphriss in https://github.com/SuperTux/supertux/pull/1856
* Textscroller Updates by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1874
* Save editor configuration to file by @Semphriss in https://github.com/SuperTux/supertux/pull/1871
* Added option to save level under a different name by @Semphriss in https://github.com/SuperTux/supertux/pull/1881
* Add delete level option to editor by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1789
* Don't allow changing select mode in object input type by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1845
* Fix snow disappearing by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1889
* Update rock throwing by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1888
* Option to set torch flame color by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1887
* Paths may now start at a user-defined node, and paths may be cloned or bound to multiple objects by @Semphriss in https://github.com/SuperTux/supertux/pull/1884
* Addons available for download in WASM builds by @Semphriss in https://github.com/SuperTux/supertux/pull/1891
* Fixed audio stop function by @Semphriss in https://github.com/SuperTux/supertux/pull/1893
* Added add-on creator by @Semphriss in https://github.com/SuperTux/supertux/pull/1890
* Fix explosion not pushing other objects by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1903
* Fix low trampoline jumps by @kmatebotond in https://github.com/SuperTux/supertux/pull/1769
* Level updates by @weluvgoatz in https://github.com/SuperTux/supertux/pull/1909
* Upgrade packages by @tobbi in https://github.com/SuperTux/supertux/pull/1913
* Update leaf_wind.stl by @JL0027 in https://github.com/SuperTux/supertux/pull/1908
* Make unstable tiles destroyable from bottom by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1916
* Add generic list selection menu and use it in an editor menu by @Rami-Slicer in https://github.com/SuperTux/supertux/pull/1895
* Added reference counter for PathObjects by @Semphriss in https://github.com/SuperTux/supertux/pull/1931
* Fix relative paths with .. in file selection by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1850
* Testing a level by pressing Ctrl+T no longer messes up the autotile config by @Semphriss in https://github.com/SuperTux/supertux/pull/1941
* Removed crash-inducing assertions with autotiles by @Semphriss in https://github.com/SuperTux/supertux/pull/1937
* Fixed some recently added tiles that were non-solid by @Semphriss in https://github.com/SuperTux/supertux/pull/1938
* Fixed GL shader compatibility (enables WebGL) by @Semphriss in https://github.com/SuperTux/supertux/pull/1948
* Fixed inconsistent jump height when jumping from a climbable by @Semphriss in https://github.com/SuperTux/supertux/pull/1947
* Rocks now apply friction when on the ground by @Semphriss in https://github.com/SuperTux/supertux/pull/1946
* Added sector bind which absence causes a crash upon resizing a sector with an offset in the editor by @Semphriss in https://github.com/SuperTux/supertux/pull/1944
* Separated GitHub workflows for macOS 10 and 11 by @Semphriss in https://github.com/SuperTux/supertux/pull/1950
* Fixed backtrack softlock on Icy Island by @Semphriss in https://github.com/SuperTux/supertux/pull/1951
* Change mrcandle to walking_candle in the dispenser menu by @Narre in https://github.com/SuperTux/supertux/pull/1953
* Newly created worldmaps use the new tileset by @Semphriss in https://github.com/SuperTux/supertux/pull/1925
* Refactored overlay_widget.cpp to improve lifetime management by @Semphriss in https://github.com/SuperTux/supertux/pull/1935
* Fix the Unknown path mode warnings by @Narre in https://github.com/SuperTux/supertux/pull/1958
* Remove unordered path mode from the editor by @Narre in https://github.com/SuperTux/supertux/pull/1957
* Refactor level flipping by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1922
* Add resources for Korean language #1885 #1597 by @gnh1201 in https://github.com/SuperTux/supertux/pull/1963
* Fixed name problem for generated AppImages by @Semphriss in https://github.com/SuperTux/supertux/pull/1962
* Fix issues with level flipping introduced in #1922 by @mrkubax10 in https://github.com/SuperTux/supertux/pull/1968
* Attempt to fix changing sprites of certain objects in editor by @Semphriss in https://github.com/SuperTux/supertux/pull/1964

## New Contributors
* @xduugu made their first contribution in https://github.com/SuperTux/supertux/pull/1409
* @szborows made their first contribution in https://github.com/SuperTux/supertux/pull/1406
* @jgarte made their first contribution in https://github.com/SuperTux/supertux/pull/1427
* @AndrePinheiroPT made their first contribution in https://github.com/SuperTux/supertux/pull/1447
* @Alasdairbugs made their first contribution in https://github.com/SuperTux/supertux/pull/1461
* @Semphriss made their first contribution in https://github.com/SuperTux/supertux/pull/1465
* @erratic-c-programmer made their first contribution in https://github.com/SuperTux/supertux/pull/1484
* @mbernard2 made their first contribution in https://github.com/SuperTux/supertux/pull/1496
* @omeritzics made their first contribution in https://github.com/SuperTux/supertux/pull/1520
* @ffontaine made their first contribution in https://github.com/SuperTux/supertux/pull/1561
* @eric135 made their first contribution in https://github.com/SuperTux/supertux/pull/1585
* @strikersh made their first contribution in https://github.com/SuperTux/supertux/pull/1602
* @ericonr made their first contribution in https://github.com/SuperTux/supertux/pull/1601
* @Raoul1808 made their first contribution in https://github.com/SuperTux/supertux/pull/1611
* @hogren made their first contribution in https://github.com/SuperTux/supertux/pull/1647
* @seanyeh made their first contribution in https://github.com/SuperTux/supertux/pull/1758
* @mrkubax10 made their first contribution in https://github.com/SuperTux/supertux/pull/1729
* @yyefymov made their first contribution in https://github.com/SuperTux/supertux/pull/1759
* @pazkero made their first contribution in https://github.com/SuperTux/supertux/pull/1765
* @jonnius made their first contribution in https://github.com/SuperTux/supertux/pull/1766
* @jwakely made their first contribution in https://github.com/SuperTux/supertux/pull/1804
* @LouriNoel made their first contribution in https://github.com/SuperTux/supertux/pull/1834
* @kmatebotond made their first contribution in https://github.com/SuperTux/supertux/pull/1769
* @JL0027 made their first contribution in https://github.com/SuperTux/supertux/pull/1908
* @Rami-Slicer made their first contribution in https://github.com/SuperTux/supertux/pull/1895
* @Narre made their first contribution in https://github.com/SuperTux/supertux/pull/1953
* @gnh1201 made their first contribution in https://github.com/SuperTux/supertux/pull/1963

**Full Changelog**: https://github.com/SuperTux/supertux/compare/v0.6.2...v0.6.3-beta.2

<hr /><em>This discussion was created from the release <a href='https://github.com/SuperTux/supertux/releases/tag/v0.6.3-beta.2'>SuperTux v0.6.3-beta.2</a>.</em>

**Comment by Rami-Slicer** [comment]
Oh my god Semphriss...

**Reply by tux314159** [reply]
Semphriss iss a god
